backend/src/agent/persistence/client/workflow_client.go

```diff
@@ -152,7 +151,7 @@ func (c *WorkflowClient) getStatusFromChildReferences(namespace, selector string
 			status.TaskRuns = taskrunStatuses
 		}
 		if hasRun {
-			runs, err := c.clientset.TektonV1alpha1().Runs(namespace).List(context.Background(), v1.ListOptions{
+			runs, err := c.clientset.TektonV1beta1().CustomRuns(namespace).List(context.Background(), v1.ListOptions{
```

Comment:
@ScrapCodes Do you think our pipelineloop custom task is ready for v1beta1? Should we keep the v1alpha1 runs as well for backward compatibility until v1alpha1 is removed? I was discussing with the Tekton community and they said v1beta1 will not have certain fields like retry until they finalized the implementation.